CHANGELOG.md

```diff
@@ -2,7 +2,7 @@
   - Add limit for cyclic arrays on Should -Be [GH-824]
   - Fix infinite recursion on Should -Be [GH-818]
   - Fix output when passing in hashtables [GH-816]
-  - Fix -Verifiable parameter on Assert-VerifiableMocks [GH-786]
+  - Fix -Verifiable parameter on Assert-VerifiableMock [GH-786]
```

Comment:
I think that for the previous releases descriptions we still should leave it as a plural.